takeshape/takeshape-starter-ecommerce-kitchen-sink
# shipping info

### Screenshots
<img width="486" alt="Screen Shot 2022-04-07 at 12 02 38" src="https://user-images.githubusercontent.com/277401/162247351-cbcc6b28-ebe4-4905-bffa-4cc992079163.png">

### Test Plan (Steps to test):

1. Review the preview site
1. Make a purchase (be sure to do a purchase with only one-time items, and a purchase that includes a subscription)
1. Review your purchases page. You should see the purchase. After a short time, you should also see the shipping details change from `processing` to `Shipped`.
2. Ensure you still get a review email (assuming you used a real email to sign up) and that your loyalty points increase.

### After merged

1. Be sure the Stripe webhook to the main site is updated to include this new hook
2. Remove the testing webhook
